### ISSUE TITLE
Questions about environment and package version.

### ISSUE THREAD

**Issue by dashpulsar** [justusschock/deep_alignment_network_pytorch#4]
I tried to run your code, but it seems incompatible. First of all, I run under python3.7: pip install deep-alignment-pytorch.
But I got the following error：
![image](https://user-images.githubusercontent.com/30170084/82585764-26ce2f80-9bc9-11ea-9c8c-c54c06092078.png)
Then I went back to 3.6 and anaconda to try to install, but it also failed.
Finally I tried to use google colab, it worked miraculously.
But I still encountered many problems, including TensorFlow version problems, path loss problems, etc., but I have solved them. But in the end, during training, I got a problem I couldn't solve:
![image](https://user-images.githubusercontent.com/30170084/82586145-afe56680-9bc9-11ea-87d8-60de9f072a74.png)
Can you give me some suggestions, I will be very grateful!

**Comment by justusschock** [comment]
I'm really sorry, you encountered theses issues. Regarding your python errors: I've never seen them and for me the setup works, sorry!

Regarding your tensorflow error: this is strange since the code is purely PyTorch so you shouldn't need tf at all.

If I find some time, I'll do a rewrite of this.
